Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -2398,6 +2391,18 @@ checkGlobFiles verbosity pkg root =
     getWarning field glob (GlobMissingDirectory dir) =
       [ PackageDistSuspiciousWarn (GlobNoDir field glob dir) ]
 
+toDependencyVersionsMap :: (PackageDescription -> [Dependency]) -> GenericPackageDescription -> Map PackageName VersionRange
```

Comment:
Why is this function at this line (i.e.: far away from where it is used, not in the `utils` part of the file)?

Reply:
I moved this to the utils section